### PR TITLE
fix(linkpanel): fixed link panel alignment.

### PR DESCRIPTION
**Pull request by michaelkleinhenz**
Fixes the link panel, making it more logical in layout and alignment. Not the best css tricks there, but works for now.

### CODE REVIEW

**Comment by fabric8cd** [comment]
@michaelkleinhenz snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1755-fabric8-planner.int.rdu2c.fabric8.io

File: src/app/work-item/work-item-detail/work-item-link/work-item-link.component.scss [hunk 1]
```diff
@@ -80,6 +88,17 @@
         @extend .flex-container;
         @extend .in-column-direction;
         padding: 0 rem(40);
+        .readonly {
+          background: white;
```
Comment (mindreeper2420) [inline src/app/work-item/work-item-detail/work-item-link/work-item-link.component.scss]: `$color-pf-white`

File: src/app/work-item/work-item-detail/work-item-link/work-item-link.component.scss [hunk 2]
```diff
@@ -80,6 +88,17 @@
...
+        }
+        .selected-value {
+          color: black !important;
```
Comment (mindreeper2420) [inline src/app/work-item/work-item-detail/work-item-link/work-item-link.component.scss]: `$color-pf-black`

File: src/app/work-item/work-item-detail/work-item-link/work-item-link.component.html [hunk 3]
```diff
@@ -45,58 +45,119 @@
...
+                </ul>
+                <!-- <span class="input-group-addon"> <span class="caret"></span> </span> -->
+              </div>
+            </div>
+          </div>
+          <div class="table-cell">
+            <button [disabled]="!selectedLinkType || !selectedWorkItemId ? true : false"
+              type="button"
+              value="Link"
+              class="btn btn-xs link-button"
```
Comment (mindreeper2420) [inline src/app/work-item/work-item-detail/work-item-link/work-item-link.component.html]: add `btn-default` for proper styling

**Pull request merged**